adhocracy4/polls/templates/a4polls/poll_detail.html

```diff
@@ -1,9 +1,9 @@
-{% extends "a4projects/project_detail.html" %}
+{% extends extends %}
 {% load react_polls react_comments %}
 
 
 {% block phase_content %}
-    <div class="module-content">
+    <div class="module-content--light">
```

Comment:
I don't think we use this class anwhere anymore but I don't think it's important

Reply:
Oh, I only tried with module-content and that looked all wrong. I didn't try without the class. Maybe we can redo it when we add it to a4 with classnames we can use by extend in both projects!